pkg-manager/resolve-dependencies/src/index.ts

```diff
@@ -125,8 +125,8 @@ export async function resolveDependencies (
     preferredVersions: opts.preferredVersions,
     virtualStoreDir: opts.virtualStoreDir,
     workspacePackages: opts.workspacePackages,
-    updateToLatest: opts.updateToLatest,
     noDependencySelectors: importers.every(({ wantedDependencies }) => wantedDependencies.length === 0),
+    // @ts-expect-error -- this fails for now
```

Comment:
this should be fixed, right?

Reply:
Ah yes, this one stumped me a bit, but I fixed it in d0560ecf9831f0139fc69b32142e5e6eb805a4c5. It seems that `injectWorkspacePakcages` is not a property of this function, but `tsc` was not complaining about the unknown property prior to my changes. Adding the `{updateToLatest?: boolean}` property prompts `tsc` to error out about `injectWorkspacePakcages`, which is what I would expect to happen in the first place. I chalked this down to `tsc` weirdness (`tsc` 5.7 did not seem to have this issue 💭)